commune_app/tests/test_vote.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from commune_app.all_models.users import User
+from commune_app.all_models.chores import Chore
+from commune_app.all_models.votes import Vote
+
+
+USER_NAME0 = "user0"
+PASSWORD0 = "123456"
+FIRST_NAME0 = "user"
+LAST_NAME0 = "usery"
+EMAIL0 = "user@gmail.com"
+ID0 = 123456789
+
+ID_NOT_IN_DB = 987654321
+
+USER_NAME1 = "user1"
+PASSWORD1 = "987654"
+FIRST_NAME1 = "user"
+LAST_NAME1 = "class"
+EMAIL1 = "userclass1@gmail.com"
+ID1 = 209461581
+
+USER_NAME = "use"
+PASSWORD = "123456789"
+FIRST_NAME = "USE"
+LAST_NAME = "MODEL"
+EMAIL = "userclass2@gmail.com"
+ID = 102030405
+
+LONG_FIELD = "A" * 151
+NOT_EMAIL = "user.com@[]"
+
+CHORE_TITLE = "Dishes"
+CHORE_DATE = (1997, 10, 19)
+CHORE_BUDGET = 30
+CHORE_DESCRIPTION = "Clean the Dishes"
+CHORE_ASSIGN = "Elias"
+CHORE_PASSED = False
```

Comment:
maybe it will be better to create it "automatically" you can add migrations files like:
https://github.com/redhat-beyond/lets-meet/blob/main/users/migrations/0002_add_user_test_data.py
and
https://github.com/redhat-beyond/lets-meet/blob/main/users/migrations/0003_createsuperuser.py
or you can open a different issue about it

Reply:
i will try to add migration files

Reply:
NP, please open an issue about it